src/scrape.py

```diff
@@ -0,0 +1,91 @@
+import requests
+import re
+from bs4 import BeautifulSoup
+
+
+def get_html_content(url):
+    response = requests.get(url)
+    html_content = response.content

```

Comment:
add validation you got a 200 status code

Reply:
for each loaded page?

Reply:
yes, you want to inform the user in case something is broken, or perhaps to retry.

You assume here that the page will always load correctly.